src/odemis/driver/simcam.py

```diff
@@ -207,8 +207,8 @@ def _setTranslation(self, value):
         """
         # compute the min/max of the shift. It's the same as the margin between
         # the centered ROI and the border, taking into account the binning.
-        max_tran = ((self._img.shape[-1] - self._resolution[0] * self._binning[0]) // 2,
-                    (self._img.shape[-2] - self._resolution[1] * self._binning[1]) // 2)
+        max_tran = ((self._img.shape[-2] - self._resolution[0] * self._binning[0]) // 2,
+                    (self._img.shape[-3] - self._resolution[1] * self._binning[1]) // 2)
```

Comment:
Why not use the following?
```suggestion
        max_tran = ((self._img.shape[0] - self._resolution[0] * self._binning[0]) // 2,
                    (self._img.shape[1] - self._resolution[1] * self._binning[1]) // 2)
```

Reply:
I was not sure if there can be more dimensions than only YXC, e.g. ZYXC.
I could change it, but then it will be like this right?
```suggestion
        max_tran = ((self._img.shape[1] - self._resolution[0] * self._binning[0]) // 2,
                    (self._img.shape[0] - self._resolution[1] * self._binning[1]) // 2)
```
Since X is on [1] and Y is on [0] index.

Reply:
That sounds like a good reason.

Reply:
So then just leave it like it was